src/App.js

```diff
@@ -1,23 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
+
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+  const [showDishes, setShowDishes] = useState(foods);
+
+  const createFood = (newFood) => {
+    const updatedFood = [newFood, ...dishes]
+    setDishes(updatedFood)
```

Comment:
Since now you have a new 'useState' that is the result of the 'Search', every time you create a new food you need to update both, so under the 'setDishes' add a line updating the 'setShowDishes' as well: 
` setShowDishes(updatedFood)`